### PR TITLE
Update id.json

### PR DESCRIPTION
**Pull request by Jironah**

### CODE REVIEW

File: web/locales/id.json [hunk 1]
```diff
@@ -562,31 +562,31 @@
...
-            "countryName": "France",
+            "countryName": "Perancis",
             "zoneName": "Corsica"
         },
         "GA": {
             "zoneName": "Gabon"
         },
         "GB": {
-            "zoneName": "Britania Raya"
+            "zoneName": "Inggris Raya"
```
Comment (jarek) [inline web/locales/id.json:575]: Hello, thank you for the changes! This seems to reference "England" more than "Britain" - is that the right reference? The wikipedia article https://id.wikipedia.org/wiki/Britania_Raya uses the Britania Raya name and seems to list Inggris, Irlandia Utara, Skotlandia, and Wales as constituents, so Inggris seems to be England?

Comment (Jironah) [reply web/locales/id.json:575]: Yup. I think that's right. 
Thank's for the correction.

Comment (skovhus) [reply web/locales/id.json]: @jarek thanks for the careful review. 👏

**Review by skovhus — APPROVED** [review]
Thanks for fixing this!

**Pull request merged**